src/index.js

```diff
@@ -33,10 +48,45 @@ mongoose
 		console.log("Database connected.");
 	});
 
+var userType = "";
+passport.serializeUser(function(userObject, done) {
+	userType = userObject.schemaType;
+	console.log(userType);
+	done(null, userObject.id);
+});
+	
+passport.deserializeUser(function(id, done) {
```

Comment:
the parameter name `done` suggests it's a boolean (to the reader).
A better name would be `onComplete`, or `resolve`. 
Note also that there is no error handling here.